utils/messages.ts

```diff
@@ -38,16 +38,16 @@ in quanto non rientri nei requisiti per il suo utilizzo.
 
 export const getMessage = (
   messageType: keyof typeof MESSAGES,
-  cgnStatus: CgnStatus
+  card: Card
 ): MessageContent => {
   switch (messageType) {
-    case "CgnRevokedStatus":
+    case "CardRevoked":
       // tslint:disable-next-line: no-useless-cast
-      return MESSAGES[messageType](cgnStatus as CgnRevokedStatus);
-    case "CgnActivatedStatus":
+      return MESSAGES[messageType](card as CardRevoked);
```

Comment:
What if `card` is not `CardRevoked`?

Reply:
It should never happens since the messageType is calculated on `Card` type.

Reply:
Ok, but if the check at the caller level is missing or wrong this function could have unexpected behaviour.